examples/pytorch/AxHyperOptimizationPTL/conda.yaml

```diff
@@ -10,8 +10,8 @@ dependencies:
       - torchvision>=0.9.1
       - torch>=1.9.0
       - protobuf<4.0.0
-      # gyptorch 1.9.0 is incompatible with the versions of botorch
+      # gyptorch 1.9.x is incompatible with the versions of botorch
```

Comment:
It looks like ax-platform is the root cause. It pins botorch to `0.8.0`. This version uses 

https://github.com/facebook/Ax/blob/9856dd4bfb3f1aa7f3b9907dd0116fd723cf3924/setup.py#L12